hms-api/src/handler/ideaComment/edit.ts

```diff
@@ -0,0 +1,17 @@
+import {buildResponse} from '../../rest/responses';
+import {wrapHandler} from '../handler-wrapper';
+import Uuid from '../../util/Uuid';
+import IdeaCommentEditRequest from '../../rest/ideaComment/IdeaCommentEditRequest';
+import {editIdeaComment} from '../../service/idea-comment-service';
+import IdeaCommentEditResponse from '../../rest/ideaComment/IdeaCommentEditResponse';
+
+export async function edit(event, context, callback) {
+  await wrapHandler(async () => {
+    const id: Uuid = event.pathParameters.id;
+    const request = IdeaCommentEditRequest.parse(event.body);
+
+    await editIdeaComment(id, request.text);
+
+    callback(null, buildResponse(200, new IdeaCommentEditResponse(id)));
+  }, callback);
+}
```

Comment:
I guess we don't really need to change it now, but there is no check in here for permission.  By that I mean we don't check that the user making the edit is the user who created the comment

Reply:
the same would also be true for the remove endpoint

Reply:
https://github.com/knoffhoff/hms/issues/1267

Reply:
awesome 👍